### PR TITLE
MC-1283: Export protos directories as cargo depvars

### PR DESCRIPTION
**Pull request by jcape**
This PR exports the `protos` directories in `attest-api` and `mobilecoin-api` as cargo dependency variables for use by downstream dependencies in their build scripts (e.g. as protoc include paths), and increments the minor version.

### CODE REVIEW

File: attest/api/Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,19 @@
 [package]
 name = "attest-api"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (eranrund) [inline attest/api/Cargo.toml:3]: 👏

File: attest/api/Cargo.toml [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 authors = ["MobileCoin"]
 license = "MIT/Apache-2.0"
 edition = "2018"
 description = "gRPC APIs for encrypted communications with an enclave"
 keywords = ["SIGMA", "Cryptography", "Key Exchange", "Diffie-Hellman", "SGX", "Attestation"]
 readme = "README.md"
+links = "mc-attest-api"
```
Comment (eranrund) [inline attest/api/Cargo.toml:10]: Why is this needed?

File: consensus/api/Cargo.toml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 [package]
 name = "mobilecoin-api"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["MobileCoin"]
 build = "build.rs"
 edition = "2018"
+links = "mc-consensus-api"
```
Comment (mfaulk) [inline consensus/api/Cargo.toml:7]: I'm not familiar with `links` here. How does this fit in to the plan?

https://doc.rust-lang.org/cargo/reference/manifest.html#the-links-field

File: attest/api/Cargo.toml (hunk 2 above)
Comment (jcape) [reply attest/api/Cargo.toml:10]: You need a `links` key in `Cargo.toml`, in order to have cargo export variables. So, you do `cargo_emit::pair!` to emit the "cargo:KEY=VALUE" stuff, and if you have a `links` key, dependents of the crate doing the emission will get `DEP_<SCREAMING_SNAKE_LINKS>_KEY=VALUE` injected into their environment.

File: consensus/api/Cargo.toml (hunk 3 above)
Comment (jcape) [reply consensus/api/Cargo.toml:7]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key

File: attest/api/Cargo.toml (hunk 2 above)
Comment (jcape) [reply attest/api/Cargo.toml:10]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key

Comment (eranrund) [reply attest/api/Cargo.toml:10]: lol SCREAMING_SNAKE_LINKS
